no-computer.md

```diff
@@ -11,34 +11,47 @@ _What need is met by your idea? Who are you helping? What is that person's
+- Badminton rackets look dumb.
+
+My DSL, pingminton, combines both sports to create a truly electrifying experience that lacks the downsides of either parent sport.  It takes the paddles from ping-pong and the birdie/net from badminton.  When playing pingminton, you don't need a table, you don't have to carry around dumb-looking badminton rackets, and (most importantly) you can whack the birdy with a large amount of force without it going too far.  This combination of ease-of-play, looking not lame, and getting to use lots of force creates the perfect environment in which to exert dominance over a peer.
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL is appropriate because it allows users to exert dominance over their peers in an enjoyable manner.  It isn't a necessary DSL, as there are many other sports that have the same function, but it's always nice to have more options when it comes to killing time with friends.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I came up with this idea many years ago with my brother.  We were in a competitive spirit and wanted to play badminton, but found that we'd always hit the birdie too far when using badminton rackets.  We tried using different objects that wouldn't send the birdie so far (books, slabs of wood, etc), and eventually settled on ping-pong paddles.  This DSL excites me because I spent a large portion of my childhood playing it.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Exert dominance over your peers
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+Users interact with the language (the sport + the opponent) by hitting the birdy over the net with their ping-pong paddle.  If the birdie returns over the net, they must continue hitting it over the net until the desired outcome is achieved (e.g. athletic dominance has been exerted).  In terms of programming, the syntax of a single hit consists of the angle of the ping-pong paddle, the speed at which contact is made, and various other factors like wind speed and the material of the paddle.
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+When the program runs (ie a game is played), both users are interacting with the program at the same time.  For each user, the program consists of the game + their opponent, and the program runs until the match has ended and one player has won.  The program interacts with the user by accepting "hits" as input, and it reports errors (such as hitting the birdie into the net) by having the opponent laugh at the player.
```

Comment:
Are these errors though? If a person hits the ball into the net, isn't that a valid command/language output?